install/installer/pkg/components/public-api-server/configmap.go

```diff
@@ -35,7 +35,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	})
 
 	cfg := config.Configuration{
-		GitpodServiceURL:               fmt.Sprintf("wss://%s/api/v1", ctx.Config.Domain),
+		GitpodServiceURL:               fmt.Sprintf("wss://%s/api/gitpod", ctx.Config.Domain),
```

Comment:
Why mthis change? More of a drive-by, or was it required for some reason? :thinking:

Reply:
The other endpoint doesn't accept cookies

Reply:
Ah, thansk for refreshing my memory.